solidity/contracts/Gravity.sol

```diff
@@ -230,14 +228,12 @@ contract Gravity is ReentrancyGuard {
 		);
 
 		// Check that new validators and powers set is well-formed
-		require(_newValidators.length == _newPowers.length, "Malformed new validator set");
+		require(_newValidators.length == _newPowers.length || _newValidators.length == 0, "Malformed new validator set");
```

Comment:
This is the first substantive change in this PR (other than the signature struct refactor). I guess it doesn't hurt but it's also not very useful. This entire require statement could probably go, and probably should. I just never got around to removing it.

Reply:
This update is related to the issue https://github.com/PeggyJV/gravity-bridge/issues/257
Should I ignore this issue? @jtremback